release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -557,6 +557,13 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-expose-mtu_{context}"]
+==== Exposing MTU for vfio-pci SR-IOV devices
+
+With this release, maximum transmission unit (MTU) on virtual function using the `vfio-pci` driver is available in the network-status pod annotation, and inside the container. 
+
+For more information, see xref:../networking/hardware_networks/nw-sriov-expose-mtu.adoc#nw-sriov-expose-mtu_configuring-sr-iov[Exposing MTU for vfio-pci SR-IOV devices to pod].
```

Comment:
I tracked down the section you're looking for, and it's still in the `add-pod.adoc` file, so that part of the xref needs to change, but otherwise I think the rest of the xref looks good and would work:
```suggestion
For more information, see xref:../networking/hardware_networks/add-pod.adoc#nw-sriov-expose-mtu_configuring-sr-iov[Exposing MTU for vfio-pci SR-IOV devices to pod].
```